assets/tests/components/map.test.tsx

```diff
@@ -209,7 +209,7 @@ describe("map", () => {
     expect(onClick).not.toHaveBeenCalled()
   })
 
-  test("renders street view link if in maps test group", async () => {
+  test("renders street view link from stop if in maps test group", async () => {
```

Comment:
praise: thanks for adding this! would you mind adding it to the test below this one as well?